services/server/server.go

```diff
@@ -65,7 +66,11 @@ type Server struct {
 	platformService      platformService
 	awsInstanceChecker   *AWSInstanceChecker
 	grafanaClient        grafanaClient
-	l                    *logrus.Entry
+	RulesService         *ia.RulesService
```

Comment:
let's keep all fields private.

Reply:
I wanted to keep them private, but that I will have to create functions to access them from outside. Do you prefer having a public function that will return the the rules service?

Reply:
what is the reason to get `rulesService` from `Server`?

Reply:
We need to call the its instance functions when deleting or regenerating the rules files: https://github.com/percona/pmm-managed/blob/PMM-7232-Disabling-IA-continues-sending/services/server/server.go#L581

Reply:
It can be private there to call.

Reply:
It then fails in main.go when registered.
![image](https://user-images.githubusercontent.com/49949480/107197189-dcf5d580-69f3-11eb-81a8-0abbd998aa31.png)

Reply:
don't get `rulesService` from `deps.server`, in main.go you have direct access to `rulesService`

Reply:
I do not see any big reason to make field public too.